src/Workspaces/Core/Portable/Classification/SyntaxClassification/AbstractNameSyntaxClassifier.cs

```diff
@@ -116,5 +117,37 @@ internal abstract class AbstractNameSyntaxClassifier : AbstractSyntaxClassifier
 
             result.Add(new ClassifiedSpan(span, ClassificationTypeNames.StaticSymbol));
         }
+
+        protected static bool IsStaticSymbol(ISymbol? symbol)
```

Comment:
this ends up duplicating all the logic (Which would make maintenance much harder).  We would want teh function above to now call this helper so we only have it in one place, supporting mutiple clients.